maven/workspace.bzl

```diff
@@ -1,11 +1,25 @@
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("//internal:maybe.bzl", "maybe")
 
-def maven(tag = "3.3", sha = "d85951a92c0908c80bd8551002d66cb23c3434409c814179c0ff026b53544dab"):
+def maven(tag = "5.3", sha = "d31e369b854322ca5098ea12c69d7175ded971435e55c18dd9dd5f29cc5249ac"):
     maybe(
         http_archive,
         name = "rules_jvm_external",
         strip_prefix = "rules_jvm_external-%s" % tag,
         sha256 = sha,
         url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % tag,
     )
+
+# Load maven rules from git
+def maven_repository(
+        remote = "https://github.com/sugarmanz/rules_jvm_external.git",
```

Comment:
plan is to try to get your PR in `rules_jvm_external` merged, yea?

Reply:
Ideally, but IMO it's not really acceptable as-is. It requires an `--experimental_google_legacy_api` flag, which would require every consumer of the rules to enable that flag. I'm not sure what the path forward is, but will likely try to get some feedback from the team over there.

Reply:
Opened a PR for feedback:
https://github.com/bazelbuild/rules_jvm_external/pull/1049